packages/List/src/List.stories.tsx

```diff
@@ -0,0 +1,219 @@
+    description: readme,
+  },
+} as ComponentMeta<typeof List>;
+
+const largeOptionList: Option[] = [
+  {
+    type: 'list',
+    label: 'Item 1',
+    value: '1',
+    description: 'just adding some description',
+  },
+  {
+    type: 'list',
+    label: 'Item 2 (disabled)',
+    value: '2',
+    disabled: true,
+  },
+  {
+    type: 'list',
+    label: 'Item 3 (focused)',
+    value: '3',
+    icon: <Happiness size="small" />,
+    description: 'just adding some description',
+  },
+  {
+    type: 'list',
+    label: 'Item 4 (selected)',
+    value: '4',
+  },
+  {
+    type: 'list',
+    label: 'Item 5',
+    value: '5',
+    color: '#74DCC9',
+  },
+  {
+    type: 'list',
+    label: 'Item 6',
+    value: '6',
+  },
+  {
+    type: 'list',
+    label: 'Item 7',
+    value: '7',
+  },
+];
+
+const premiumOptionlist: Option[] = [
+  {
+    type: 'list',
+    label: 'Item 1',
+    value: '1',
+    description: 'just adding some description',
+  },
+  {
+    type: 'list',
+    label: 'Item 2',
+    value: '2',
+  },
+  {
+    type: 'list',
+    label: 'Item 3 (Premium)',
+    value: '3',
+    icon: <Happiness size="small" />,
+    premium: true,
+  },
+  {
+    type: 'list',
+    label: 'Item 4',
+    value: '4',
+  },
+];
+
+const membersList: Member[] = [
+  {
+    type: 'member',
+    member: 'List item with click and hover events',
+    value: '1',
+  },
+  {
+    type: 'member',
+    member: 'Item 2',
+    value: '2',
+    role: 'just adding some description',
+    src: 'https://i.pravatar.cc/100',
+    manager: true,
+  },
+  {
+    type: 'member',
+    member: 'Item 3 (focused)',
+    value: '3',
+  },
+  {
+    type: 'member',
+    member: 'Item 4 (selected)',
+    value: '4',
+  },
+  {
+    type: 'member',
+    member: 'Item 5',
+    value: '5',
+  },
+  {
+    type: 'member',
+    member: 'Item 6',
+    value: '6',
+    src: 'https://i.pravatar.cc/100',
+  },
+  {
+    type: 'member',
+    member: 'Item with icon',
+    value: 'icon',
+    src: 'https://i.pravatar.cc/100',
+  },
+];
+
+const Template: ComponentStory<typeof List> = (args) => {
+  const [selectedOption, setSeletedOption] = React.useState<OptionType | null>(
+    largeOptionList[3]
+  );
+  const [focusedOption, setFocusedOption] = React.useState<OptionType | null>(
+    largeOptionList[2]
+  );
+
+  function handleOptionSelect(option: OptionType) {
+    setSeletedOption(option);
+  }
+
+  function handleOptionFocus(option: OptionType) {
+    setFocusedOption(option);
+  }
+
+  return (
+    <List
+      {...args}
+      onOptionChange={handleOptionSelect}
+      onOptionFocus={handleOptionFocus}
+      selectedOption={selectedOption}
+      focusedOption={focusedOption}
+      style={{ maxWidth: '40rem' }}
+    />
+  );
+};
+export const Overview = Template.bind({});
+
+Overview.args = {
+  options: largeOptionList,
+};
+
+export const Members = Template.bind({});
+
+Members.args = {
+  options: membersList,
+};
+
+export const NotCompact = Template.bind({});
```

Comment:
switch for render a compact list here and a default list on Overview

```suggestion
export const Compact = Template.bind({});
```

Reply:
The default list is compact. We came to this decision after speaking with Francis about it.